dbms/src/Storages/MutableSupport.h

```diff
@@ -14,84 +14,53 @@
 
 #pragma once
 
-#include <Common/typeid_cast.h>
-#include <Core/Block.h>
 #include <Core/Names.h>
-#include <DataTypes/DataTypeNullable.h>
-#include <DataTypes/DataTypesNumber.h>
-#include <IO/WriteHelpers.h>
+#include <DataTypes/IDataType.h>
+#include <Storages/KVStore/Types.h>
 
 #include <ext/singleton.h>
 
 namespace DB
 {
-class MutableSupport : public ext::Singleton<MutableSupport>
+class MutSup : public ext::Singleton<MutSup>
 {
 public:
-    MutableSupport()
+    MutSup()
     {
         mutable_hidden.push_back(version_column_name);
         mutable_hidden.push_back(delmark_column_name);
         //mutable_hidden.push_back(tidb_pk_column_name);
-
-        all_hidden.insert(all_hidden.end(), mutable_hidden.begin(), mutable_hidden.end());
     }
 
     const OrderedNameSet & hiddenColumns(const String & table_type_name)
     {
-        if (mmt_storage_name == table_type_name || txn_storage_name == table_type_name
-            || delta_tree_storage_name == table_type_name)
+        if (delta_tree_storage_name == table_type_name)
             return mutable_hidden;
         return empty;
     }
 
-    static const String mmt_storage_name;
-    static const String txn_storage_name;
-    static const String delta_tree_storage_name;
-
-    static const String tidb_pk_column_name;
-    static const String version_column_name;
-    static const String delmark_column_name;
-    static const String extra_table_id_column_name;
-
-    static const DataTypePtr tidb_pk_column_int_type;
-    static const DataTypePtr tidb_pk_column_string_type;
-    static const DataTypePtr version_column_type;
-    static const DataTypePtr delmark_column_type;
-    static const DataTypePtr extra_table_id_column_type;
-
-    /// mark that ColumnId of those columns are defined in dbms/src/Storages/KVStore/Types.h
-
-    // TODO: detail doc about these delete marks
-    struct DelMark
-    {
-        static const UInt8 NONE = 0x00;
-        static const UInt8 INTERNAL_DEL = 0x01;
-        static const UInt8 DEFINITE_DEL = (INTERNAL_DEL << 1);
-        static const UInt8 DEL_MASK = (INTERNAL_DEL | DEFINITE_DEL);
-        static const UInt8 DATA_MASK = ~DEL_MASK;
-
-        static UInt8 getData(UInt8 raw_data) { return raw_data & DATA_MASK; }
-
-        static bool isDel(UInt8 raw_data) { return raw_data & DEL_MASK; }
+    inline static const String delta_tree_storage_name = "DeltaMerge";
 
-        static bool isDefiniteDel(UInt8 raw_data) { return raw_data & DEFINITE_DEL; }
+    inline static constexpr ColumnID tidb_pk_col_id = -1;
```

Comment:
```suggestion
    inline static constexpr ColumnID extra_handle_id = -1;
```
"pk_col_id" is not accurate. Align with the name "extra_handle_id" with that in tidb-server?
https://github.com/pingcap/tidb/blob/d13e52ed6e22cc5789bed7c64c861578cd2ed55b/pkg/meta/model/table.go#L31-L38